.github/workflows/third-party-library-changes.yml

```diff
@@ -0,0 +1,63 @@
+name: Third Party Library Changes
+
+on:
+  pull_request:
```

Comment:
Should we also run this on `push` just as a sanity check? In case someone is not using GitHub to stage changes and run tests?